docs/design/network-discovery/README.md

```diff
@@ -1,149 +1,149 @@
-process is not some one time action. We must be able to track and react on
-topology change. I.e. new inter switch links (ISL) must be detected and used 
-as possible route for existing and future flows. Corrupted (temporary or
-permanently) ISLs must be detected too. All flows must be evacuated from such
-ISLs.
-
-Discovered ISL consist of:
-* datapath + port-number (source)
-* datapath + port-number (dest)
-* link-speed
-* link-latency
-* BFD-support (and other optional capabilities)
-
-Open-Kilda produce "discovery" packet (Ethernet+IP+UDP+LLDP), put inside source
-datapath, source port-number, current time and send it using PACKET_OUT OF
-message to the source switch and set as PORT_OUT source port-number in actions.
-
-All switches have OF rule that match and send to controller our "discovery"
-packets (using PACKET_IN message.
-
-When controller receive "discovery" packet via PACKET_IN message it extract
-source datapath and source port-number extract datapath and port-number 
-(from PACKET_IN message) and treat it as destination datapath and destination
-port-number. So now it have both ISL endpoints. Link speed is extracted from
-current port-speed. Latency is calculated as current time minus packet create
-time (extracted from packet) minus source switch latency (calculated by FL
-using echo-request/response OF messages) minus dest switch latency. All
-accumulated data used to create "discovery-event" and pass in into storm for
-further processing.
-
-This "process" is done for each enabled port of each switch on each N 
-seconds(configuration parameter). As result we will receive 2 discovery
-packets/event for each "link" - one for each direction. And because it repeats
-periodically Open-Kilda can "detect" ISL-fails i.e. link-corruptions. And react
+One of the main Open-Kilda tasks is the automatic network discovery. This discovery 
+process is not a one-time action. We must be able to track and react on
+topology changes. For example, a new inter-switch link (ISL) must be detected and used 
+as a possible route for existing or future flows. Temporary or permanently corrupted 
+ISLs must be detected and all flows must be evacuated from these kind of ISLs.
+
+Information about a discovered ISL contains:
+* data path + port number (source)
+* data path + port number (dest)
+* link speed
+* link latency
+* whether it supports BFD or other optional capabilities
+
+Open-Kilda installs on all switches an OF rule that manages discovery packets. Switches use
+that rule to send and receive discovery packets (based on PACKET_IN and PACKET_OUT messages),
+as well as to report the state to the controller.
+
+Open-Kilda produces a "discovery" packet (Ethernet+IP+UDP+LLDP), puts inside it a source
+data path, source port number, current time. Open-Kilda sends this packet using a PACKET_OUT OF
+message to the source switch and sets the source port number in actions as PORT_OUT.
```

Comment:
Please, use OpenKilda instead of Open-Kilda as in the other parts

Reply:
I'll update all 5 occurrences